Formula/mdcat.rb

```diff
@@ -1,10 +1,10 @@
 class Mdcat < Formula
   desc "Show markdown documents on text terminals"
-  homepage "https://codeberg.org/flausch/mdcat"
-  url "https://codeberg.org/flausch/mdcat/archive/mdcat-0.28.0.tar.gz"
+  homepage "https://github.com/swsnr/mdcat"
+  url "https://github.com/swsnr/mdcat/archive/refs/tags/mdcat-0.30.3.tar.gz"
```

Comment:
- https://codeberg.org/flausch/mdcat goes to 404
- https://codeberg.org/flausch user account redirects to https://codeberg.org/swsnr
- https://codeberg.org/swsnr/mdcat is archived with reference to https://github.com/swsnr/mdcat

Also matches URLs used by official crate: https://crates.io/crates/mdcat